### PR TITLE
Require Symfony ^4.4

### PR DESCRIPTION
**Pull request by pamil**

### CODE REVIEW

**Comment by pamil** [comment]
No idea why PHP 7.2 application build fails :/

**Comment by oallain** [comment]
> No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received

I feel like it's just a lag time that has lasted too long (more than 10 minutes).

I think relaunching the build can validate (or not) this idea ;)

File: src/Sylius/Bundle/OrderBundle/Tests/DependencyInjection/SyliusOrderExtensionTest.php [hunk 1]
```diff
@@ -28,17 +29,19 @@ final class SyliusOrderExtensionTest extends AbstractExtensionTestCase
      */
     public function it_autoconfigures_cart_contexts_and_order_processors(): void
     {
+        $this->markTestSkipped('Symfony 4.4 makes it impossible to test autoconfigured services this way.');
```
Comment (lchrusciel) [inline src/Sylius/Bundle/OrderBundle/Tests/DependencyInjection/SyliusOrderExtensionTest.php]: Maybe we should just remove this test then.

**Comment by lchrusciel** [comment]
Thanks, Kamil! :1st_place_medal:

**Pull request merged**